src/ansys/geometry/core/designer/design.py

```diff
@@ -166,6 +172,11 @@ def beam_profiles(self) -> list[BeamProfile]:
         """List of beam profile available for the design."""
         return list(self._beam_profiles.values())
 
+    @property
+    def parameters(self) -> list[Parameter]:
+        """List of parameters available for the design."""
+        return self.get_all_parameters()
+
```

Comment:
Nice wrapping - users will expect a "parameters" attribute for sure.

Reply:
yeah done this way since it seemed safer to get them on the spot in case they are updated behind the scene. hope it is ok

Reply:
Agreed yeah - seems like a safe approach!